packages/react-server/src/ReactServerStreamConfigBrowser.js

```diff
@@ -49,7 +49,9 @@ export function writeChunk(
     if (__DEV__) {
       if (precomputedChunkSet?.has(chunk)) {
         console.error(
-          'A precomputed chunk was passed to writeChunk without being copied, please use .slice() to copy it first.',
+          'A large precomputed chunk was passed to writeChunk without being copied.' +
+            ' Large chunks get enqueued directly and are not copied however this is incompatible with precomputed chunks because you cannot enqueue the same precomputed chunk twice.' +
```

Comment:
>Large chunks get enqueued directly and are not copied however this is incompatible

This looks like a run-on sentence to me.

Reply:
Split up the sentences, hopefully it's clearer.